README.md

```diff
@@ -14,36 +14,133 @@
 
-As of version `3.0.0`, the internal data model was adjusted to allow CycloneDX VEX documents to be produced as per
-[official examples](https://cyclonedx.org/capabilities/bomlink/#linking-external-vex-to-bom-inventory) linking VEX to a separate CycloneDX document.
+## Installation
 
-If you're looking for a CycloneDX tool to run to generate (SBOM) software bill-of-materials documents, why not checkout 
-[CycloneDX Python][cyclonedx-python] or [Jake][jake].
+**Via pip:**
+```shell
+pip install cyclonedx-python-lib
+```
 
-## Documentation
-
-View the documentation [here](https://cyclonedx-python-library.readthedocs.io/).
+**Via Conda:**
+```shell
+conda install -c conda-forge cyclonedx-python-lib
+```
 
 ## Python Support
 
-We endeavour to support all functionality for all [current actively supported Python versions](https://www.python.org/downloads/).
+We endeavor to support all functionality for all [current actively supported Python versions](https://www.python.org/downloads/).
 However, some features may not be possible/present in older Python versions due to their lack of support.
 
-## Changelog
+## VEX Support
+
+As of version `3.0.0`, the library supports CycloneDX VEX documents production with [official example](https://cyclonedx.org/capabilities/bomlink/#linking-external-vex-to-bom-inventory) compatibility for linking VEX to separate CycloneDX documents.
+
+## Documentation
 
-See our [CHANGELOG][chaneglog_file].
+Complete documentation is available on [Read the Docs][link_rtfd]. This includes:
+- API Reference
+- Usage Examples
+- Integration Guides
+- Best Practices
+
+## Responsibilities
+
+* Provide a general-purpose Python implementation of [CycloneDX][link_website]
+* Provide type hints for implementation support
+* Support JSON/XML document parsing and generation
+* Validate CycloneDX documents against schema specifications
+* Support multiple CycloneDX specification versions
+* Maintain comprehensive data models for BOM manipulation
+* Enable pip-based installation for downstream usage
+
+## Capabilities
+
+### Enums
+* `BomFormat` - BOM format types
+* `ComponentType` - Types of components (e.g., APPLICATION, LIBRARY)
+* `ComponentScope` - Component scope types
+* `DataFlow` - Data flow types
+* `Encoding` - Encoding types
+* `ExternalReferenceType` - Types of external references
+* `HashAlgorithm` - Supported hash algorithms
+* `ImpactAnalysisAffectedStatus` - Impact analysis affected status types
+* `ImpactAnalysisJustification` - Impact analysis justification types
+* `ImpactAnalysisResponse` - Impact analysis response types
+* `ImpactAnalysisState` - Impact analysis state types
+* `IssueClassification` - Issue classification types
+* `LifecyclePhase` - Lifecycle phase types
+* `PatchClassification` - Patch classification types
+* `VulnerabilityScoreSource` - Vulnerability score source types
+* `VulnerabilitySeverity` - Vulnerability severity types
+
+### Data Models
```

Comment:
could you explain the idea behind this grouping? how is this organized, how came this together? 
I really love to understand your reasoning. You might have fresh ideas that are beneficial for the project :-) please share your thoughts.



I mean, all these models were based of [CycloneDX spec](https://cyclonedx.org/specification/overview/) -- they actually reflect it.
If a grouping was actually necessary, it should correlate with the CycloneDX spec groups somehow.